pkg/framework/aries/framework.go

```diff
@@ -373,6 +382,19 @@ func createOutboundDispatcher(frameworkOpts *Aries) error {
 	return nil
 }
 
+func createMessenger(frameworkOpts *Aries) error {
+	ctx, err := context.New(
```

Comment:
Since messenger is something which is built on top of OutboundDispatcher, both shouldn't co-exist in context.

Reply:
We need both at least for now. As soon as everything in the framework will be replaced with the messenger usage it can be removed.